libs/experimental/tests/unit_tests/test_data_anonymizer.py

```diff
@@ -78,6 +94,8 @@ def test_add_recognizer_operator() -> None:
     assert anonymized_text == "<TITLE> Jane Doe was here."
 
     # anonymizing with custom recognizer and operator
+    anonymizer = PresidioAnonymizer(analyzed_fields=[])
+    anonymizer.add_recognizer(custom_recognizer)
```

Comment:
is it necessary here?

Reply:
Actually, not in this place, only in reversible one. Thanks :smile: